lib/controllers/user.controller.js

```diff
@@ -0,0 +1,62 @@
+import UserService from '../services/user.service';
+import CommonValidator from '../validators/common.validator';
+import UserValidator from '../validators/user.validator';
+
+export default class UserController {
+    constructor() {
+        this.userService = new UserService();
+        this.commonValidator = new CommonValidator();
+        this.userValidator = new UserValidator();
+    }
+
+    find(req, res, next, user_id) {
```

Comment:
I would suggest to name this function like `findById`. Generally, `find` means that you are finding multiple items.

Reply:
Done